src/Uri.php

```diff
@@ -307,4 +307,26 @@ private static function rawurlencodeMatchZero(array $match): string
     {
         return \rawurlencode($match[0]);
     }
+
+    private static function safeStrtolower(string $string): string
```

Comment:
Same as https://github.com/Nyholm/psr7/pull/131/files#r359853348
` strtr($host, 'ABCDEFGHIJKLMNOPQRSTUVWXYZ', 'abcdefghijklmnopqrstuvwxyz');` does the same but is much faster